src/test/java/uk/gov/hmcts/reform/civil/notification/EvidenceUploadRespondentNotificationHandlerTest.java

```diff
@@ -79,13 +63,7 @@ void shouldNotifyRespondent2Solicitor_whenInvoked() {
                 .atStateClaimDetailsNotified().addRespondent2(YesOrNo.YES)
                 .respondentSolicitor2EmailAddress("respondentsolicitor@example.com")
                 .build();
-            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData)
-                .request(CallbackRequest.builder()
-                             .eventId(NOTIFY_RESPONDENT_SOLICITOR2_FOR_EVIDENCE_UPLOAD.name()).build()).build();
-
-            //when: about to submit is called for callback handler with respondent2 event
-            handler.handle(params);
-
+            handler.notifyRespondentEvidenceUpload(caseData, false);
```

Comment:
//when: Evidence for Respondent Solicitor 2 is uploaded

Reply:
done